addons/web/static/src/js/core/popover.js

```diff
@@ -145,9 +145,9 @@ odoo.define('web.Popover', function (require) {
                 this.popoverRef.el.classList.add(`o_popover--${position.name}`);
             } else {
                 // use the given `position` props because no position fits
-                this.popoverRef.el.style.top = `${positioningData[this.props.position].top}px`;
-                this.popoverRef.el.style.left = `${positioningData[this.props.position].left}px`;
-                this.popoverRef.el.classList.add(`o_popover--${this.props.position}`);
+                this.popoverRef.el.style.top = `${positioningData['bottom'].top}px`;
+                this.popoverRef.el.style.left = `${positioningData['bottom'].left}px`;
+                this.popoverRef.el.classList.add(`o_popover--bottom`);
```

Comment:
This is a problem actually, for example in the case of emoji, if the screen is really too small, it will go bottom but it's imposible to scroll down the chat window